.circleci/build.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -ex
+
+export MAX_JOBS=8
+
+PYTORCH_DIR=/tmp/pytorch
```

Comment:
If this is running in a VM, make sure you have enough sized tmpfs.
In my workstation between pytorch and pytorch/xla there is 6GB of stuff (source, build, ...).

Reply:
Sure! We use the same type of VM as used in Pytorch CI, which I think should be enough. If not, we can figure something out to get a larger VM. :D Thanks!